src/main/scala/com/mongodb/spark/Logging.scala

```diff
@@ -16,74 +16,9 @@
 
 package com.mongodb.spark
 
-import org.slf4j.{Logger, LoggerFactory}
-
 /**
- * Utility trait for classes that want to log data. Creates a SLF4J logger for the class and allows
- * logging messages at different levels using methods that only evaluate parameters lazily if the
- * log level is enabled.
+ * Utility abstract class implementation of Logging.
+ *
+ * Allows Java consumers to use the underlying Logging trait
  */
-private[spark] trait Logging {
```

Comment:
Traits with implementation can't be used by Java classes - that is why this moved round.